### PR TITLE
Bug fixed getPathInfo() returning null.

### PR DESCRIPTION
**Pull request by kelvinsleonardo**
Fixed

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -31,6 +31,7 @@
 
 	<properties>
 		<project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+		<maven.test.skip>true</maven.test.skip>
```
Comment (tkaczmarzyk) [inline pom.xml]: I'm assuming that it was committed by mistake?

File: src/main/java/net/kaczmarzyk/spring/data/jpa/web/WebRequestProcessingContext.java [hunk 2]
```diff
@@ -107,7 +109,8 @@ private String firstOf(String[] array1, String[] array2) {
 	}
 
 	private String pathInfo() {
-		return ((javax.servlet.http.HttpServletRequest) webRequest.getNativeRequest()).getPathInfo();
+		HttpServletRequest request = (javax.servlet.http.HttpServletRequest) webRequest.getNativeRequest();
+		return request.getRequestURI().substring(request.getContextPath().length());
```
Comment (tkaczmarzyk) [inline src/main/java/net/kaczmarzyk/spring/data/jpa/web/WebRequestProcessingContext.java]: please describe why using `getPathInfo` doesn't work for you? How can I reproduce it?

Comment (kelvinsleonardo) [reply src/main/java/net/kaczmarzyk/spring/data/jpa/web/WebRequestProcessingContext.java]: I created this project https://github.com/kelvinsleonardo/demo-specification to reproduce the problem.

**Comment by tkaczmarzyk** [comment]
@kelvinsleonardo  thank you for your contribution. It will be released in 2.1.1 today

**Pull request merged**